### PR TITLE
[DNM] Toggleable antag faction

### PR DESCRIPTION
**Pull request by AllCorruptingWyrm**
## About The Pull Request

After discussions about the Minutemen idea with Frok and other staff, I asked Frok if I could do an antag faction that can be enabled slot-wise with weapons and sprites to boot so, I'm doing just that with frok saying like around four slots so sure.

## Why It's Good For The Game

Flirtation with the concept of antag factions, probably not gonna work but hey we have zetans too.

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog

:cl:
add: Added new things
add: Added more things
/:cl:

### CODE REVIEW

**Comment by chemistrymain2** [comment]
by antag faction, what i meant was like, a random event firing off to make people said thing by opening ghost spawners in said base

**Comment by AllCorruptingWyrm** [comment]
> by antag faction, what i meant was like, a random event firing off to make people said thing by opening ghost spawners in said base

yeah just making this abundantly clear for everyone involved by the way****************

File: code/__DEFINES/jobs.dm [hunk 1]
```diff
@@ -190,6 +191,11 @@
 #define F13FOLLOWERVOLUNTEER	(1<<3)
 #define F13PROFESSOR	(1<<4)
 
+#define INSTITUTE (1<<10)
```
Comment (BadAtThisGame302) [inline code/__DEFINES/jobs.dm:194]: You defined this wrong, it's meant to be 1<<12 and the roles 1<<0... 1<<1 etc.

File: code/game/objects/items/devices/radio/headset.dm [hunk 2]
```diff
@@ -375,9 +376,21 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	. = ..()
 	AddComponent(/datum/component/wearertargeting/earprotection, list(SLOT_EARS))
 
+/obj/item/radio/headset/headset_enclave/ComponentInitialize()
```
Comment (BadAtThisGame302) [inline code/game/objects/items/devices/radio/headset.dm:379]: This is pointless, since there's this exact same thing above it.

Comment (BadAtThisGame302) [reply code/game/objects/items/devices/radio/headset.dm:379]: Unless you mean to do this for the institute radio below it, if so my mistake.

**Comment by AllCorruptingWyrm** [comment]
fucked some shit up so i'm going to take a crack at this later.

**Pull request closed without merging**